command/api/token/create.go

```diff
@@ -80,14 +82,23 @@ func createAction(ctx *cli.Context) (err error) {
 		return fmt.Errorf("team-id argument must be a valid UUID")
 	}
 
+	aud := ctx.String("audience")
+	switch aud {
+	case "":
+	case "step-agent":
+	default:
+		return fmt.Errorf("invalid audience %q", aud)
+	}
```

Comment:
I would remove this for the moment. We haven't released it, and the backend should ignore an unknown or misconfigured one.

Reply:
b03e9130